docs/create-edit-dashboards/add-dashboard.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Add visualizations to dashboard
+nav_order: 4
+parent: Creating and Editing Dashboards (Panels)
```

Comment:
Can we rename it to just `Panels` to avoid long names?

Reply:
Well, that's the thing. Its not fully about Panels....its more about using the dashboard which includes using and making visualizations.

Just `Panels` would not do I think